core/src/components/select/select.ionic.scss

```diff
@@ -9,14 +9,17 @@
   --background: #{globals.$ion-primitives-base-white};
   // TODO(ROU-10778, ROU-10875): Sync the color names to the design system of
   // ios and md. This will allow us to have a single color map.
-  --border-color: #{globals.current-color(neutral)};
+  --border-color: #{globals.$ion-primitives-neutral-500};
```

Comment:
Based on the designs, it seems that they're using neutral-800.

![Screenshot 2024-12-16 at 10 44 37 AM](https://github.com/user-attachments/assets/216901c9-ae4f-4108-afd2-f7718a3f138f)

Reply:
That's true, no idea where I got the 500 when inspecting the Figma file 🤔 .

Will change, thanks for the heads-up!

Reply:
Perfect! That makes sense to me. Thank you for the double check!!